packages/cli/commands/user-management/reset.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import Command from '@oclif/command';
+import { Not } from 'typeorm';
+import { LoggerProxy } from '../../../workflow/dist/src';
+import { Db } from '../../src';
+import { getLogger } from '../../src/Logger';
+
+export class Reset extends Command {
+	static description = '\nResets the database to the default user state';
+
+	async run() {
+		const logger = getLogger();
+		LoggerProxy.init(logger);
+		await Db.init();
+
+		try {
+			const globalRole = await Db.collections.Role!.findOne({
+				name: 'owner',
+				scope: 'global',
+			});
+
+			const instanceOwner = await Db.collections.User!.findOneOrFail({ globalRole });
+
+			// switch all workflows ownership to owner
+			await Db.collections.SharedWorkflow!.update(
+				{ user: Not(instanceOwner) },
+				{ user: instanceOwner },
+			);
+
+			// switch all credentials ownership to owner
+			await Db.collections.SharedCredentials!.update(
+				{ user: Not(instanceOwner) },
+				{ user: instanceOwner },
+			);
+
+			// delete all users from users table except owner
+			await Db.collections.User!.delete({ id: Not(instanceOwner.id) });
+
+			// reset user to being a shell
+			await Db.collections.User!.save(
+				Object.assign(instanceOwner, {
+					firstName: null,
```

Comment:
The initial migration sets first and last names to `default`. I think we should remain consistent, changing either here or the initial migration.